api/cpp/esp-idf/slint/idf_component.yml

```diff
@@ -0,0 +1,7 @@
+# Copyright © SixtyFPS GmbH <info@slint.dev>
+# SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+description: "Slint — declarative GUI toolkit"
+url: "https://slint.dev"
+dependencies:
```

Comment:
AFAICS this currently requires freertos as depending component.

Reply:
I don't think that's necessary because it's implied by idf  (i think)